src/weapons/Skin.java

```diff
@@ -3,97 +3,40 @@
 import java.util.ArrayList;
 import java.util.List;
 
+import lombok.Getter;
+import lombok.Setter;
+import lombok.EqualsAndHashCode;
+
+@Getter
+@Setter
+@EqualsAndHashCode
 public class Skin {
 	private String name;
 	private List<String> weapons = new ArrayList<String>();
 	private int id;
 	private String tag;
 	private String lang;
 	
-	public String getName() {
-		return name;
-	}
-	
-	public void setName(String name) {
-		this.name = name;
-	}
-	
-	public List<String> getWeapons() {
-		return weapons;
-	}
-	
-	public int getId() {
-		return id;
-	}
-	
-	public void setId(int id) {
-		this.id = id;
-	}
-	
-	public String getTag() {
-		return tag;
-	}
-	
-	public void setTag(String tag) {
-		this.tag = tag;
+	private String capitalize(String str) {
+		return str.substring(0, 1).toUpperCase() + str.substring(1);
 	}
 	
 	public String getLang() {
-		if (name.contains("phase1"))
-		{
-			return lang + " (Phase 1)";
-		}
-		else if (name.contains("phase2"))
-		{
-			return lang + " (Phase 2)";
-		}
-		else if (name.contains("phase3"))
-		{
-			return lang + " (Phase 3)";
-		}
-		else if (name.contains("phase4"))
-		{
-			return lang + " (Phase 4)";
-		}
-		else if (name.contains("emerald_marbleized"))
-		{
-			return lang + " (Emerald)";
-		}
-		else if (name.contains("ruby_marbleized"))
-		{
-			return lang + " (Ruby)";
-		}
-		else if (name.contains("sapphire_marbleized"))
-		{
-			return lang + " (Sapphire)";
-		}
-		else if (name.contains("blackpearl_marbleized"))
-		{
+		if (name.contains("phase)) {
+			return lang + " (" + capitalize(name.substring(0, 1)) + " " + name.charAt(5) + ")";
+		} else if (name.contains("marbleized") && !name.contains("blackpearl")) {
+			return lang + " (" + capitalize(name.replace("marbleized", "")) + ")";
+		} else if (name.contains("blackpearl_marbleized")) {
```

Comment:
I don't think this change is making it more readable. I'd rather stick with the long if else than do string operations that is hard to read and understand what it actually does. Simple is better